k8s.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: git2grid
+  namespace: git2grid
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  minReadySeconds: 5
+  template:
+    metadata:
+      labels:
+        app: git2grid
+        role: git2grid
+    spec:
+      containers:
+        - name: git2grid
+          image: billytrend/git2grid:latest
```

Comment:
Instead of using `:latest`, I'd pin to a particular tagged version. It'll make rolling out updates smoother in the future.